tmvis.js

```diff
@@ -1802,7 +1805,7 @@ TimeMap.prototype.filterMementos = function(response, curCookieClientId, callbac
         else {
             currentTime = new Date(t.mementos[i]["datetime"].split(",")[1]).getTime();
             daysBetween = (currentTime - lastTime) / (1000*60*60*24);
-            if(daysBetween >= 3) { //If mementos are at least 3 days apart
+            if(daysBetween >= 3) { //If mementos are at least 7 days apart
```

Comment:
The value of days between is 3, but the comment says 7. Is this how it should be?

Reply:
The comment is supposed to say 3. I forgot to pull this [commit](https://github.com/a-mabe/tmvis/commit/c2e031aa6dd8f7cb8c69c97506cb513dc94de8d1) before I made this PR.